HISTORY.md

```diff
@@ -1,4 +1,8 @@
 # Rocksdb Change Log
+## Unreleased
+### Bug Fixes
+* Fixed the logic of populating native data structure for `read_amp_bytes_per_bit` during OPTIONS file parsing.
```

Comment:
Is this line referring to just this fix? If so, I think we can say it was a fix for big endian only (little endian not broken).

If it's referring to #7659 also, I think there should be more detail about what could go wrong.

Reply:
Updated.